iConverter/Pages/Base/Router.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Router.swift
+//  iConverter
+//
+//  Created by Aram Semerjyan on 13.04.22.
+//
+
+import UIKit
+
+class Router {
+
+    deinit {
+        print("********")
+        print("Router deinited")
+        print("********")
+    }
+
+    private var topViewController: UIViewController? {
+        let keyWindow = UIApplication.shared.windows.first(where: { $0.isKeyWindow })
+
+        if var topController = keyWindow?.rootViewController {
+            while let presentedViewController = topController.presentedViewController {
+                topController = presentedViewController
+            }
+
+            return topController
+        }
+
+        return nil
+    }
+
+    func setAndPresentRootController(_ controller: UIViewController,
```

Comment:
I know you copied this from Paysera source code but please fix indentation of parameters (next line)